woocommerce-myparcel.php

```diff
@@ -118,17 +120,9 @@ private function getVersion(): string
          * @return void
          * @throws \Exception
          */
-        private function setupWebhooks(): void
+        private function registerWebhooks(): void
         {
-            $changeOrderStatusAfter = WCMP_Export_Consignments::getSetting(
-                WCMYPA_Settings::SETTING_CHANGE_ORDER_STATUS_AFTER
-            );
-            $exportMode             = WCMP_Export_Consignments::getSetting(WCMYPA_Settings::SETTING_EXPORT_MODE);
-
-            if (WCMP_Settings_Data::CHANGE_STATUS_AFTER_PRINTING === $changeOrderStatusAfter && WCMP_Settings_Data::EXPORT_MODE_PPS === $exportMode) {
-                (new OrderStatusWebhook())->register();
-            }
-
+            (new OrderStatusWebhook())->register();
```

Comment:
Dit ziet er goed uit, dat je deze rest-url altijd aanmaakt, alleen dan verwacht ik wel de check die gedaan werd, of er wel iets geautomatiseerd moet worden, in de orderstatuswebhook, en daar zie ik geen wijzigingen in deze pr, ik heb nu dus het idee dat de status altijd gezet wordt, ongeacht de instellingen in de plugin. Graag uitleg waarom dat niet zo is, of het fixen :-)